spec/models/forms/condition_computer_spec.rb

```diff
@@ -27,13 +27,15 @@
         "integer",
         ["integer", "integer", %w[integer integer]],
         "integer",
-        "integer" # Hidden question
+        "integer", # Hidden question
+        "integer" # Disabled question
       ])
     end
     let(:form_items) { computer.preordered_form_items }
 
     before do
-      form.c[7].update!(hidden: true)
+      form.c[6].update!(hidden: true)
+      form.c[7].update!(disabled: true)
```

Comment:
Are you sure you're disabling the right question to match the data structure above? Seems you added another question above as the disabled one, so it should be `c[8]`, whilst `c[7]` stays as the hidden one?